SwiftGit2.xcodeproj/project.pbxproj

```diff
@@ -729,7 +729,7 @@
 			);
 			runOnlyForDeploymentPostprocessing = 0;
 			shellPath = /bin/sh;
-			shellScript = "if which swiftlint >/dev/null; then\n    swiftlint --lenient\nelse\n    echo \"warning: SwiftLint is not installed\"\nfi";
+			shellScript = "if which swiftlint >/dev/null; then\n    swiftlint\nelse\n    echo \"warning: SwiftLint is not installed\"\nfi";
```

Comment:
Like this
<img width="1440" alt="screen shot 2017-04-27 at 07 31 39" src="https://cloud.githubusercontent.com/assets/8848587/25469326/dba3a92a-2b1b-11e7-9a12-220a89a84352.png">

And this is my version
<img width="377" alt="screen shot 2017-04-27 at 07 34 16" src="https://cloud.githubusercontent.com/assets/8848587/25469341/f4293dfc-2b1b-11e7-9dbc-d0960854da43.png">